pytripgui/canvas_vc/plotter/coordinate_info.py

```diff
@@ -0,0 +1,103 @@
+import numpy as np
+from matplotlib.projections import register_projection
+from mpl_toolkits.mplot3d import Axes3D
+
+
+class CoordinateInfo(Axes3D):

```

Comment:
Add a comment describing responsibility of this class

Reply:
Done